bundles/net.tourbook/src/net/tourbook/ui/views/rawData/DialogReimportTours.java

```diff
@@ -60,57 +72,64 @@
+   private static final String             STATE_IS_IMPORT_POWER_AND_SPEED              = "STATE_IS_IMPORT_POWER_AND_SPEED";              //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_RUNNING_DYNAMICS             = "STATE_IS_IMPORT_RUNNING_DYNAMICS";             //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_SWIMMING                     = "STATE_IS_IMPORT_SWIMMING";                     //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_TEMPERATURE                  = "STATE_IS_IMPORT_TEMPERATURE";                  //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_TRAINING                     = "STATE_IS_IMPORT_TRAINING";                     //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_TOUR_MARKERS                 = "STATE_IS_IMPORT_TOUR_MARKERS";                 //$NON-NLS-1$
+   private static final String             STATE_IS_IMPORT_TIMER_PAUSES                 = "STATE_IS_IMPORT_TIMER_PAUSES";                 //$NON-NLS-1$
+   private static final String             STATE_IS_SKIP_TOURS_WITH_IMPORTFILE_NOTFOUND = "STATE_IS_SKIP_TOURS_WITH_IMPORTFILE_NOTFOUND"; //$NON-NLS-1$
 
-   private static final int             VERTICAL_SECTION_MARGIN                      = 10;
+   private static final int                VERTICAL_SECTION_MARGIN                      = 10;
 
-   private static final IDialogSettings _state                                       = TourbookPlugin.getState("DialogReimportTours"); //$NON-NLS-1$
+   private static final IDialogSettings    _state                                       = TourbookPlugin.getState("DialogReimportTours"); //$NON-NLS-1$
 
-   private final ITourViewer3           _tourViewer;
+   private static CountDownLatch           _countDownLatch;
 
-   private SelectionAdapter             _defaultListener;
+   private static ArrayBlockingQueue<Long> _dbUpdateQueue;
 
-   private PixelConverter               _pc;
+   private static ThreadPoolExecutor       _dbUpdateExecutor;
 
+   private static ThreadFactory            _threadFactory;
+   private final ITourViewer3              _tourViewer;
+
+   private SelectionAdapter                _defaultListener;
+
+   private PixelConverter                  _pc;
    //
-   private Image _imageLockClosed = CommonActivator.getThemedImageDescriptor(CommonImages.Lock_Closed).createImage();
-   private Image _imageLockOpen   = CommonActivator.getThemedImageDescriptor(CommonImages.Lock_Open).createImage();
+   private Image                           _imageLockClosed                             = CommonActivator.getThemedImageDescriptor(
+         CommonImages.Lock_Closed).createImage();
+   private Image                           _imageLockOpen                               = CommonActivator.getThemedImageDescriptor(
+         CommonImages.Lock_Open).createImage();
 
```

Comment:
Reformatting issue also here.

The trick here is that, an empty line and `//` comment start a new indention for the fields and they do not wrap

![image](https://user-images.githubusercontent.com/1283445/126028551-b9f7a306-59c4-4041-b7a8-3c37bb96d993.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/127576142-205d74ae-b6ee-4272-aa8a-55bdc071c064.png)